webrtc.html

```diff
@@ -1015,29 +1015,51 @@
+                reason, then user agent MUST queue a task that runs the following
                 steps:</p>
                 <ol>
                   <li>
                     <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
                     <code>true</code>, then abort these steps.</p>
                   </li>
                   <li>
-                    <p>If elements of the SDP were modified, then reject
-                    <var>p</var> with a newly
-                    <a data-link-for="exception" data-lt="create">created</a>
-                    <code>InvalidModificationError</code>
-                    and abort these steps.</p>
-                  </li>
-                  <li>
                     <p>If the <var>description</var>'s <code><a data-for=
                     "RTCSessionDescription">type</a></code> is invalid for the
-                    current <a>signaling state</a> of <var>connection</var>,
+                    current <a>signaling state</a> of <var>connection</var>
+                    as described in <span data-jsep=
+                    "processing-a-local-desc processing-a-remote-desc">[[!JSEP]]</span>,
                     then reject <var>p</var> with a newly
                     <a data-link-for="exception" data-lt="create">created</a>
                     <code>InvalidStateError</code> and abort these steps.</p>
                   </li>
                   <li>
+                    Let <var>lastOffer</var> be the value of 
+                    <code><a>RTCSessionDescriptionInit</a>.sdp</code> returned
+                    by the last call to <code>createOffer</code>.
+                  </li>
+                  <li>
+                    Let <var>lastAnswer</var> be the value of
+                    <code><a>RTCSessionDescriptionInit</a>.sdp</code> returned
+                    by the last call to <code>createAnswer</code>.
+                  </li>
+                  <li>
+                    If <code><var>description</var>.type</code> is
+                    <code>offer</code> and <code><var>description</var>.sdp</code>
+                    is not equal to <var>lastOffer</var>, then reject
+                    <var>p</var> with a newly <a data-link-for="exception"
+                    data-lt="create">created</a>
+                    <code>InvalidModificationError</code> and abort these steps.
+                  </li>
+                  <li>
+                    If <code><var>description</var>.type</code> is
+                    <code>answer</code> or <code>pranswer</code> and
+                    <code><var>description</var>.sdp</code> is not equal
+                    to <var>lastAnswer</var>, then reject <var>p</var>
+                    with a newly <a data-link-for="exception"
+                    data-lt="create">created</a>
+                    <code>InvalidModificationError</code> and abort these steps.
+                  </li>
```

Comment:
This set of steps should only apply when setting a local description; `InvalidModificationError` shouldn't be thrown when setting a remote offer that differs from `lastOffer`.